cmd/webhook/main.go

```diff
@@ -77,6 +78,11 @@ var types = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 	net.SchemeGroupVersion.WithKind("ServerlessService"): &net.ServerlessService{},
 }
 
+var callbacks = map[schema.GroupVersionKind]validation.Callback{
+	v1.SchemeGroupVersion.WithKind("Service"): validation.NewCallback(
+		extravalidation.ExtraServiceValidation, webhook.Create, webhook.Update),
```

Comment:
One can create config directly, so you need to validate that as well.

Reply:
I'm trying to ensure this works with Service first with all the e2e tests. I'll put another callback directly on Configuration as a follow-up